src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -169,7 +169,7 @@ private void Analyze(TOptions options, AggregatingLogger logger)
             targets = ValidateTargetsExist(_rootContext, targets);
 
             // 5. Initialize report file, if configured.
-            InitializeOutputFile(options, _rootContext, targets);
```

Comment:
>InitializeOutputFile(options, _rootContext, targets); [](http://example.com/codeflow?start=12&length=53)

we won't initialize the outputfile with targets because we don't want to emit all targets in the artifacts.